terraform/apps/observability/terraform.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 1.8.1"
+  required_providers {
+    docker = {
+      source  = "kreuzwerker/docker"
+      version = "~> 3.0"
+    }
+  }
+}
+
+provider "docker" {
+  host = "unix:///var/run/docker.sock"
```

Comment:
I have a Windows workstation and a problem with this host: `Error initializing Docker client: protocol not available` when calling `terraform plan`.

But `npipe:////.//pipe//docker_engine` works as a `host` for me. Maybe you can also use it, and this code can be more platform-agnostic.

Reply:
Okay, I will try to use WSL2 for working with Terraform.